x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -273,10 +279,13 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
     initServer(libs);
 
     core.getStartServices().then(([_, depsStart]) => {
+      const eqlSearchStrategy = securityEqlSearchStrategyProvider(this.logger);
       const securitySolutionSearchStrategy = securitySolutionSearchStrategyProvider(depsStart.data);
       const securitySolutionTimelineSearchStrategy = securitySolutionTimelineSearchStrategyProvider(
         depsStart.data
       );
+
+      plugins.data.search.registerSearchStrategy(SECURITY_EQL_SEARCH_STRATEGY, eqlSearchStrategy);
```

Comment:
If the strategy is owned by the data plugin, it should be registered there too

Reply:
@lizozom since EQL is only available in x-pack, should this strategy live in the `data_enhanced` plugin for now?